demo/demo_image_repo.py

```diff
@@ -26,7 +26,6 @@
 """
 from __future__ import print_function
 from __future__ import unicode_literals
-from io import open
```

Comment:
I think there other modules that import io.open(): demo_primary.py and demo_secondary.py
Should we open an issue for this?